### DISCUSSION TITLE
Display or variable for world age (Time Played in Statistics * 24 for hours)

### DISCUSSION THREAD

**Discussion by tetraDB** [maruohon/minihud#185] [Q&A]
I've tried to find an existing way to display the Time Played from statistics (in hours) along with the {DAY} in world time info, but I have failed. I tried looking through the code, but quickly got lost in the source.

Is it already possible to show the Time Played from statistics (preferably in hours [normally  Time Played * 24h]) or may I ask for a feature/enhancement to make that available in the info -> dataFormatMinecraft so it could be configured to show something like
World Age {TIMEPLAYED * 24} or World Age {TIMEPLAYED_HRS}

any enlightenment or consideration would be much appreciated, thank you.

**Comment by maruohon** [comment]
The only thing that exists for this currently is the info line for total world time in ticks. I assume the statistics page also probably uses that? I'll need to check... But since it's a separate info line, and there is no way to do airthmetic with them currently, you unfortunately can't get it on the same line as the world time atm and converted to the desired format. I guess I could add a new "variable"/placeholder for it for now, but there is also going to be a total rewrite of how the info lines work. They will all become fully customisable with these placeholder variables, and also most likely with support for doing basic arithmetic with them.

**Reply by tetraDB** [reply]
Oh ok great, thank you for the reply and consideration!
A different line would have worked too, but it would definitely have needed converting from ticks to hours for it to make any sense.
This is really more relevant to hardcore players where the hours they survive has the most comparable meaning, days are really skewed by sleeping, but it's not a big deal also converting the days from the stats page :)
